### PR TITLE
points!

### PR DESCRIPTION
**Pull request by hingler**
- tally up goo count based on incoming messages
- reward messages with a high slime content
- reward longer messages

### CODE REVIEW

File: src/main/java/org/gooteam/balance/BalanceTicker.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                balanceDelta
+        );
+
+
+        logger.info("uptick balance for ID {}: {} cm3", userId, balanceDelta);
+    }
+
+    private double computeMessageWeight(String message) {
+        // goes nuts for links
+        double lengthWeight = Math.clamp(Math.log(Math.max(message.length(), 1)), 0.1, 8.0);
```
Comment (jasobeck) [inline src/main/java/org/gooteam/balance/BalanceTicker.java]: Is there a way to not count links towards the lengthWeight? or maybe give them the same weight as like a 4 letter word

**Review by jasobeck — COMMENTED** [review]
line 51. ( idk how to use this comment feature so idk if this comment is necessary)

File: src/main/java/org/gooteam/balance/BalanceTicker.java (hunk 1 above)
Comment (hingler) [reply src/main/java/org/gooteam/balance/BalanceTicker.java]: that makes sense - i think there's a link parser that'll work in java?

Comment (hingler) [reply src/main/java/org/gooteam/balance/BalanceTicker.java]: went w using a dumb regex expression to remove http/https links

**Comment by hingler** [comment]
btw did branch protection get undone? says i can merge as is but gonna wait for x2 check

**Review by jasobeck — APPROVED** [review]
looks good :3

**Pull request merged**